jme3-glsl-highlighter/src/com/jme3/gde/glsl/highlighter/GlslLanguage.java

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I know it's annoying, but a License Header as described in #155 would be nice to have. It's only a 5 minute task